docs/new.sh

```diff
@@ -10,7 +10,7 @@ function new() {
 		return 1
 	fi
 	local next
-	next=$(LC_ALL=C tr -dc 'a-z0-9' < /dev/urandom | head -c5)
+	next=$(LC_ALL=C tr -dc '0-9' < /dev/urandom | head -c6)
```

Comment:
I'm curious about the reason for this change. Could you please add some context?

Reply:
the non-numerical IDs look like garbled words,. my brain keeps trying to parse them which trips me up ever so slightly each time I glance at the url.

Basically: nitpicking :)